src/StatusQ/Components/StatusCommunityCard.qml

```diff
@@ -0,0 +1,289 @@
+
+   Example of how the component looks like:
+   \image status_community_card.png
+   Example of how to use it:
+   \qml
+        StatusCommunityCard {
+            locale: "en"
+            communityId: model.communityId
+            loaded: model.available
+            logo: model.icon
+            name: model.name
+            description: model.description
+            members: model.members
+            popularity: model.popularity
+            categories:  model.categories
+
+            onCardSelected: { d.navigateToCommunity(communityId) }
+        }
+   \endqml
+   For a list of components available see StatusQ.
+*/
+Rectangle {
+    id: root
+
+    /*!
+       \qmlproperty string StatusCommunityCard::communityId
+       This property holds the community identifier value.
+    */
+    property string communityId: ""
+    /*!
+       \qmlproperty bool StatusCommunityCard::loaded
+       This property holds a boolean value that represents if the community information is loaded or not.
+    */
+    property bool loaded: true
+    /*!
+       \qmlproperty string StatusCommunityCard::logo
+       This property holds the community logo source.
+    */
+    property string logo: ""
+    /*!
+       \qmlproperty string StatusCommunityCard::name
+       This property holds the community name.
+    */
+    property string name: ""
+    /*!
+       \qmlproperty string StatusCommunityCard::description
+       This property holds the community description.
+    */
+    property string description: ""
+    /*!
+       \qmlproperty int StatusCommunityCard::members
+       This property holds the community members value.
+    */
+    property int members: 0
+    /*!
+       \qmlproperty int StatusCommunityCard::popularity
+       This property holds the community popularity (community rate).
+    */
+    property int popularity: 0
+    /*!
+       \qmlproperty string StatusCommunityCard::categories
+       This property holds the data that will be populated as the tags row of the community.
+
+       Here an example of the model roles expected:
+       \qml
+            categories: ListModel {
+                ListElement { name: "gaming"; emoji: "🎮"; selected: false}
+                ListElement { name: "art"; emoji: "🖼️️"; selected: false}
+                ListElement { name: "crypto"; emoji: "💸"; selected: false}
+                ListElement { name: "nsfw"; emoji: "🍆"; selected: false}
+                ListElement { name: "markets"; emoji: "💎"; selected: false}
+            }
+       \endqml
+    */
+    property ListModel categories: ListModel {}
+    /*!
+       \qmlproperty string StatusCommunityCard::locale
+       This property holds the application locale used to give format to members number representation.
+       If not provided, default value is "en".
+    */
+    property string locale: "en"
```

Comment:
Is this needed? I thought applications would be decide what the overall locale is? How would this behave if an application is set to `es` but this component receives (for whatever reason) `en`?

We don't have `locale` properties in any of the other components, so I'm wondering why this is needed now

Reply:
Here we have 2 options:
- Exposing a more generic `string` property instead of assuming the `members` representation is a number, then the formatting of the members number is a complete responsability of the consumer.
- Exposing the `locale`, as it is currently implemented, then the formatting of the members number depends on this locale but it is done in the component implementation. 

At the end, what you said about "the application would be decide what the overall locale is" is true in both options since it is a parameter and you set it from your application consumer, the component does not decide from you. And in both options, if the component receives a wrong format (in case of exposing locale, another different from the global, or in case of direct format, a number format that is not aligned with the global locale), the behavior of the component will be the same so it will be visualized depending on the consumer input.

The response to "why this is needed now" and "why we do not have other components using it".. I'm not sure if we have a specific case where we are representing just only `numbers` in a component. Maybe our components are always more generic and a field is not restricted to a `number` bc it can really be a whatever.
Maybe the implementation of the `card` component here is not as generic as it should be and I should think of it as another more generic component with properties like this:
- Instead of `name` --> `title`
- Instead of `description` --> `subtitle`
- Instead of `members` --> `titleAsideText`
- Exposing an `icon` property instead of hardcoding the `tiny-contact` icon.
...

What are your thoughts?

Reply:
Ah I see! I didn't realize the locale was used primarily for the member count. I guess this is to ensure we get either sth. like `1,000` vs `1000` vs `1.000` ?

Either way, I think then it's fine as it is. Probably not necessary to make it too generic as, by definition, that label is supposed to be a number.

Reply:
Yeah! That's exactly for that!